_help/index.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
This needs some refinement, and needs to have `published: false` removed from it when that work is in